scaleway/resource_instance_server.go

```diff
@@ -202,6 +203,34 @@ func resourceScalewayInstanceServer() *schema.Resource {
 					Type: schema.TypeString,
 				},
 			},
+			"private_network": {
+				Type:        schema.TypeList,
+				Optional:    true,
+				MaxItems:    8,
```

Comment:
Precise it in the documentation that we cannot have an instance in more than 8 different private networks.